frontend/src/app/data-entry/source/source.component.scss

```diff
@@ -0,0 +1,4 @@
+.entity-item:not(:last-child)::after {
+    // Character \00A0 is a non-breaking space.
+    content: ",\00A0";
+}
```

Comment:
Oh neat, this is much cleaner than my solution! :)

Perhaps we could generalise this into a more general rule that we can reuse in the application? E.g. `.inline-list > .inline-list-item`

Reply:
Good idea!